### ISSUE TITLE
Custom BEM syntax

### ISSUE THREAD

**Issue by gunnx** [brigade/scss-lint#492]
It is great to see BEM being included but wondered if it was open to having custom formats.

I've been using this format in my project:

.block-test__element-test--modifier-test { }

Might also be good to add some examples around what hyphenated and strict BEM examples look like as its not that clear on the bem.info site

Thank you

**Comment by gunnx** [comment]
Sorry I see it does this already!

**Comment by brunowego** [comment]
@gunnx how you fix it?

I get this linter message too.

```
Selector `i__inbox` should be written in lowercase with hyphens (SelectorFormat)
```

**Comment by brunowego** [comment]
@gunnx you disable it using `.scss-lint.yml` right?

```yml
linters:
  SelectorFormat:
    enabled: false
```

**Comment by brunowego** [comment]
Sorry @gunnx, I find the solution. Thanks

https://github.com/brigade/scss-lint/blob/master/config/default.yml
